src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+/**
+ *
+ */
+public class Block {
```

Comment:
This class lacks header comments. You shouldn't leave blank header comments, instead, follow the javadoc format listed in the coding standard.

Same for the other classes.